jsonHttpHandler/main.go

```diff
@@ -71,6 +72,22 @@ func (h JsonHttpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
     }
   }()
 
+  ctx := r.Context()
```

Comment:
Since the request payloads can be really read only once after which they appear empty, I'm reading it and storing in Context as a string upon every request.

Reply:
Are you going to implement `TeeReader` solution in the end?